automated/linux/video/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM python:3
+
+COPY requirements.txt /home/
+RUN python3 -m pip install opencv-python scikit-image requests
```

Comment:
why don't you have all this in requirements.txt and install from there?

Reply:
It takes 1.5 hours. Both Dockerfile and requirements.txt are not needed for the test. They're added if someone would like to a) build the container to run the test in LAVA, b) run the test locally.

Reply:
ah, and using requirments.txt in Dockerfile will be tricky. If the requirements.txt changes, the layer cache will not be invalidated.